src/services/uploader/uploading.controller.test.ts

```diff
@@ -137,4 +145,82 @@ describe('UploadingController', () => {
       expect(upload).toBeTruthy()
     });
   });
+
+  describe('Start encode', () => {
+    it('Should encode successfully', async () => {
+      const jwtToken = 'test_jwt_token';
+
+      await linkedAccountsRepository.linkHiveAccount('singleton/starkerz/hive', 'starkerz');
+
+      const upload = await uploadingService.createUpload({ sub: 'singleton/starkerz/hive', username: 'starkerz', id: 'test_id' });
+
+      process.env.DELEGATED_ACCOUNT = 'threespeak'
+
+      return request(app.getHttpServer())
+        .post('/v1/upload/start_encode')
+        .set('Authorization', `Bearer ${jwtToken}`)
+        .send({ 
+          username: 'starkerz',
+          upload_id: upload.upload_id,
+          video_id: upload.video_id,
+          permlink: upload.permlink
+        })
+        .expect(201)
+        .then(response => {
+          expect(response.body).toEqual({});
+        });
+    })
+
+    it('Should fail if the upload details are falsified', async () => {
+      const jwtToken = 'test_jwt_token';
+
+      await linkedAccountsRepository.linkHiveAccount('singleton/starkerz/hive', 'sisygoboom');
+
+      process.env.DELEGATED_ACCOUNT = 'threespeak'
```

Comment:
Same as previous comments
```suggestion
```

Reply:
the whole point of this test is to test linked accounts, notice starkerz's sub and my username

Reply:
Oops misread "sisygoboom" as "starkerz"